client/handlers_test.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ClientHandlerTestSuite struct {
+	suite.Suite
+	*require.Assertions
+}
+
+func (s *ClientHandlerTestSuite) SetupTest() {
+	s.Assertions = require.New(s.T())
+}
+
+func (s *ClientHandlerTestSuite) TestCompareTags_Empty() {
+	match := compareTags("", "")
+	require.Equal(s.T(), true, match, "compareTags should return ture when fed with two empty strings")
```

Comment:
The `ClientHandlerTestSuite` inherits from the `require.Assertions`, therefore the function must use it's own receivers exports e.g.:
`s.Equal(true, match, "compareTags should return ture when fed with two empty strings")`

Reply:
For `Equal(true, ...` matchers there's a built'n forwarded matcher `True(...)`